.github/workflows/deploy.yml

```
@@ -8,47 +10,97 @@ on:
+
+      - name: Build and Push Docker Image (Production)
+        uses: docker/build-push-action@v5
         with:
-          service: careercopilot-backend-prod
-          project_id: ${{ secrets.GCP_PROJECT_ID }}
-          credentials: ${{ secrets.GCP_SA_KEY }}
-          region: us-central1
-          source: ./backend
+          context: ./backend
+          push: true
+          tags: australia
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)